rdrf/rdrf/templatetags/has_subscribable_email.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+from rdrf.models.definition.models import EmailNotification
+
+register = template.Library()
+
+
+# check whether the registry the user belongs to has subscribable email notifications.
```

Comment:
Comment here is a bit redundant. For TRRF we generally don't add comments except to explain unintuitive, unusual, or complex behavior. Nor do we use Python docstrings etc.